tracer.go

```diff
@@ -139,7 +139,7 @@ func (tracer *tracerImpl) Inject(sc ot.SpanContext, format interface{}, carrier
 	switch format {
 	case ot.TextMap, ot.HTTPHeaders:
 		return theTextMapPropagator.Inject(sc, carrier)
-	case BinaryCarrier:
```

Comment:
To make this backward compatible, can we make `const BinaryCarrier = ot.Binary` at `propagation_binary.go:12` too?